osidb/tests/test_flaw.py

```diff
@@ -923,55 +923,60 @@ def test_validate_cvss3_model(self, cvss3, should_alert):
         assert should_alert == ("cvss3_missing" in FlawFactory(cvss3=cvss3)._alerts)
 
     @pytest.mark.parametrize(
-        "summary,is_major_incident,req,should_raise",
+        "summary,is_major_incident,req,should_alert,alerts",
         [
-            ("", True, None, True),
-            ("", True, "+", True),
-            ("", True, "?", True),
-            ("", True, "-", False),
-            ("", False, None, False),
-            ("", False, "+", False),
-            ("", False, "?", False),
-            ("", False, "-", False),
-            ("foo", False, None, False),
-            ("foo", False, "+", False),
-            ("foo", False, "?", False),
-            ("foo", False, "-", False),
-            ("foo", True, None, True),
-            ("foo", True, "+", False),
-            ("foo", True, "?", True),
-            ("foo", True, "-", False),
+            ("", True, None, True, ["mi_summary_missing", "mi_summary_not_reviewed"]),
+            ("", True, "+", True, ["mi_summary_missing"]),
+            ("", True, "?", True, ["mi_summary_missing", "mi_summary_not_reviewed"]),
+            ("", True, "-", False, []),
+            ("", False, None, False, []),
+            ("", False, "+", False, []),
+            ("", False, "?", False, []),
+            ("", False, "-", False, []),
+            ("foo", False, None, False, []),
+            ("foo", False, "+", False, []),
+            ("foo", False, "?", False, []),
+            ("foo", False, "-", False, []),
+            ("foo", True, None, True, ["mi_summary_not_reviewed"]),
+            ("foo", True, "+", False, []),
+            ("foo", True, "?", True, ["mi_summary_not_reviewed"]),
+            ("foo", True, "-", False, []),
         ],
     )
     def test_validate_major_incident_summary(
-        self, summary, is_major_incident, req, should_raise
+        self, summary, is_major_incident, req, should_alert, alerts
     ):
         """
         Test that a Flaw that is Major Incident has a summary
         """
         flaw1 = FlawFactory.build(
             summary=summary,
             is_major_incident=is_major_incident,
+            embargoed=False,
```

Comment:
I wonder why is this important. If the reason is that the test was giving you an error that the user is missing some LDAP group it should rather be solved by adding `embargo_access` fixture.

Reply:
The `Embargoed` field in `FlawFactory` determines the `Source` field. If `Embargoed` is `True`, `Source` will be private (this is an original case). 

This source is then checked in `_validate_private_source_no_ack`. However, this checks an acknowledgement only in `FlawMeta`, not in `FlawAcknowledgement`. If I create a flaw with a private source and with an acknowledgement via `FlawAcknowledgement` (as `FlawMeta` will be deprecated), this test fails.
The problem is that we forgot to update `_validate_private_source_no_ack` to check also `FlawAcknowledgement.`

I see 3 possible solutions how to fix this problematic test:
1. Use `Embargoed=False`, so this problem does not happen (I did this one)
2. Use `FlawMetaFactory` of the `ACKNOWLEDGMENT` type so `_validate_private_source_no_ack` passes
3. Use `FlawAcknowledgement` and update `_validate_private_source_no_ack` to check also `FlawAcknowledgement` so `_validate_private_source_no_ack` passes.

@osoukup @Elkasitu Which solution do you prefer? I prefer option 3.

Reply:
I am sorry I did not point this out from the start.

Reply:
Thank you for the explanation! Then any solution sounds OK. Probably 1 or 2 as 3 sounds out of scope of this PR.

Reply:
I updated the test as described in option 3. Please check if it is ok.

Reply:
It is all OK. Thank you